### PR TITLE
Allow force-exposing messages that we want included in documentation

### PR DESCRIPTION
**Pull request by loganintech**
Webhooks being the primary example

### CODE REVIEW

File: example/webhooks/v1/request.proto [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+package webhooks.v1;
+
+import "apigw/v1/apigw.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "gitlab.com/ductone/c1/pkg/pb/c1/webhooks/v1beta";
+
+message Body {
+  option (apigw.v1.message).message_options = {force_expose: true};
```
Comment (pquerna) [inline example/webhooks/v1/request.proto:11]: shouldn't this be a Webhook annotation?  because in the OAI spec, there are webhook specific sections right?

Comment (pquerna) [reply example/webhooks/v1/request.proto:11]: like might expect something more like this:
```
option (apigw.v1.message).message_options = {webhook_request: "NameOfWebhook"};
```

```
option (apigw.v1.message).message_options = {webhook_response: "NameOfWebhook"};
```

And this generates a named webhook block in the OpenAPI Spec?

Comment (loganintech) [reply example/webhooks/v1/request.proto:11]: Neat, let me look into how this would change to do that.

Comment (loganintech) [reply example/webhooks/v1/request.proto]: Is this along the lines of what you had in mind? We sorta have this unified webhook experience that I don't think fit well with an openapi spec. Like we can set the name but is that name valuable? https://github.com/ductone/protoc-gen-apigw/pull/50/commits/10f64b671fe80a963937d6aeb1a05ce3fadebd5e

File: example/webhooks/v1/request.proto [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+  // The type of event that triggered this Webhook.
+  //
+  // This value will match the "Webhook-Event" header.
+  //
+  // The value will be one of:
+  // - "c1.webhooks.v1.PayloadTest"
+  // - "c1.webhooks.v1.PayloadPolicyApprovalStep"
+  // - "c1.webhooks.v1.PayloadPolicyPostAction"
+  // - "c1.webhooks.v1.PayloadProvisionStep"
```
Comment (pquerna) [inline example/webhooks/v1/request.proto:42]: for purposes of this example, these should all have the `webhooks.v1` prefix , not a c1 prefix right?

File: example/bookstore/v1/bookstore_grpc.pb.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-grpc v1.4.0
+// - protoc-gen-go-grpc v1.3.0
```
Comment (pquerna) [inline example/bookstore/v1/bookstore_grpc.pb.go:3]: @loganintech your local protoc-gen-go is outdated

**Review by pquerna — APPROVED** [review]

**Pull request merged**